ComfortZoneApp/app/src/main/java/com/example/comfortzone/FlightsAdapter.java

```diff
@@ -47,6 +47,11 @@ public void addAll(List<WeatherData> cities) {
         notifyDataSetChanged();
     }
 
+    public void filterClearAndAdd(List<WeatherData> cities) {
```

Comment:
we should keep the naming consistent, this was called "weathers" from the caller, but cities here

Also, the method naming is a bit confusing, how about "updateCityList"?